src/css/adoc/table.css

```diff
@@ -204,10 +204,41 @@ div.panels table {
     border-top: unset;
     display: block;
     font-size: 14px;
-    margin-top: var(--lg);
+    margin-top: var(--lg) 0 0;
+    max-width: fit-content;
     overflow: auto;
     width: 100%;
 
+    &.grid-none {
+      & > * > tr {
+        border: none !important;
+
+        & > td {
+          border: none;
+        }
+      }
+    }
+
+    &.grid-rows {
+      td {
+          border: 1px;
+        }
+    }
+
+    &.grid-cols {
+      & tr+tr {
+        border-top: unset;
+      }
+      
+      & td+td {
+        border-left: 1px solid var(--aluminum-4);
+      }
+    }
+
+    &.frame-show {
+      border: 1px solid var(--aluminum-4);
+    }
```

Comment:
consider sorting these alphabetically to make it easier to find. In this case, frame-show show be higher than the grid-* ones